src/proc_image_processing/cpu/server/filter_factory.cc

```diff
@@ -128,20 +127,19 @@ std::unique_ptr<Filter> FilterFactory::createInstance(const std::string &name, c
 	else if(name == "VampireTorpedoesDetector"){
 		return std::move(std::make_unique<VampireTorpedoesDetector>(globalParams));
 	}
-	else if(name == "WhiteFilter"){
-		return std::move(std::make_unique<WhiteFilter>(globalParams));
-	}
-	else if(name == "WhiteNoiseTakedown"){
-		return std::move(std::make_unique<WhiteNoiseTakedown>(globalParams));
```

Comment:
maybe review the indentation of the factory generated by the script ?

Reply:
indentation is good, refactor is not